### ISSUE TITLE
Exporting to Release mode doesn't go after the first screen

### ISSUE THREAD

**Issue by salvob41** [notapixelstudio/superstarfighter#269]
Error Generated: 
```
OpenGL ES 3.0 Renderer: GeForce MX150/PCIe/SSE2
ERROR: _load: Condition ' !f ' is true. returned: ERR_CANT_OPEN
   At: scene/resources/dynamic_font.cpp:149
ERROR: _load: Condition ' !f ' is true. returned: ERR_CANT_OPEN
   At: scene/resources/dynamic_font.cpp:149
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
ERROR: Signal 'play_song' is already connected to given method 'song_is_playing' in that object.
   At: core/object.cpp:1476
I'll be handling everything from now on. I am the master
Starting game...
FORCE SAVE is True - if true the user file will be overwritten
We are going to save here: C:/Users/user/AppData/Roaming/Godot/app_userdata/superstarfighter/savegame.save this JSON
{/root/global:{enable_analytics:False, unlocked_species:{mantiacs_1:True, mantiacs_2:True, robolords_1:True, robolords_2:False, takonauts_1:True, takonauts_2:False, toriels_1:True, toriels_2:False, trixens_1:True, trixens_2:True}}}
Successfully load the game
This song is playing: Lobby5
I am Lobby and This song is Lobby5 titled Lobby5
bgm. -> Null
We are going to save here: C:/Users/username/AppData/Roaming/Godot/app_userdata/superstarfighter/savegame.save this JSON
{/root/global:{enable_analytics:False, unlocked_species:{mantiacs_1:True, mantiacs_2:True, robolords_1:True, robolords_2:False, takonauts_1:True, takonauts_2:False, toriels_1:True, toriels_2:False, trixens_1:True, trixens_2:True}}}
ERROR: emit_signal: Error calling method from signal 'entered': 'Control(MainScreen.gd)::_on_TitleScreen_entered': Method not found.
   At: core/object.cpp:1238
```

**Comment by salvob41** [comment]
plus we need to clean up the prints and debug messages :)

**Comment by salvob41** [comment]
One of the error is here: https://www.reddit.com/r/godot/comments/b3s998/condition_area_in_e_is_true_error/

**Comment by salvob41** [comment]
Error fixed in #294
